src/contract/compiler.ts

```diff
@@ -0,0 +1,133 @@
+/*
+ * ISC License (ISC)
+ * Copyright (c) 2022 aeternity developers
+ *
+ *  Permission to use, copy, modify, and/or distribute this software for any
+ *  purpose with or without fee is hereby granted, provided that the above
+ *  copyright notice and this permission notice appear in all copies.
+ *
+ *  THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES WITH
+ *  REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF MERCHANTABILITY
+ *  AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR ANY SPECIAL, DIRECT,
+ *  INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES WHATSOEVER RESULTING FROM
+ *  LOSS OF USE, DATA OR PROFITS, WHETHER IN AN ACTION OF CONTRACT, NEGLIGENCE OR
+ *  OTHER TORTIOUS ACTION, ARISING OUT OF OR IN CONNECTION WITH THE USE OR
+ *  PERFORMANCE OF THIS SOFTWARE.
+ */
+
+/**
+ * ContractCompilerHttp module
+ *
+ * @module @aeternity/aepp-sdk/es/contract/compiler
+ * @export ContractCompilerHttp
+ * @example import { ContractCompilerHttp } from '@aeternity/aepp-sdk'
+ */
+
+import { RestError } from '@azure/core-rest-pipeline'
+import stampit from '@stamp/it'
+import semverSatisfies from '../utils/semver-satisfies'
+import { Compiler as CompilerApi } from '../apis/compiler'
+import { UnsupportedVersionError } from '../utils/errors'
+
+interface ErrorFormatterBody {
+  reason?: object
+  info?: object
+  parameter?: object
+}
+interface ErrorFormatterError {
+  message: string
+  pos: {col: number, line: number}
+  type: string
+  context?: string
+}
+
+export abstract class _ContractCompilerHttp {
```

Comment:
```suggestion
export class _ContractCompilerHttp {
```
it doesn't seems to have abstract methods